### DISCUSSION TITLE
Give all entities the ability to pierce living things.

### DISCUSSION THREAD

**Discussion by Trseeds** [studio-minus/ppg-bugs-and-requests#2197] [Ideas]
Just an idea.

**Comment by ghost** [comment]
how

**Reply by Trseeds** [reply]
I don't know, it's just an idea.

**Reply by ghost** [reply]
i think you need to explain the idea

**Comment by Trseeds** [comment]
> Just an idea.

Example, Chuck the barrel of a shotgun towards a human, the shotgun with then glide through like the sword does.

**Comment by dinevolence** [comment]
That's ripping through the human with brute force, not to mention that it could potentially break the game, it would also probably be very experimental. Also you said entities, which would be living organisms.
